### DISCUSSION TITLE
PopOS on ASUS ROG Zephyrus G14 Laptop

### DISCUSSION THREAD

**Discussion by jptraverso26** [abskmj/notes#23] [General]
I receive this error:

Err:11 https://download.opensuse.org/repositories/home:/luke_nukem:/asus/xUbuntu_20.04  Release
  404  Not Found [IP: 195.135.223.226 443]

**Comment by abskmj** [comment]
This package is needed only if you are using an older version (20.10 and older) of PopOS.
